packaging/Linux/albatross_log.service

```
@@ -4,17 +4,18 @@
 # systemctl edit albatross_log.service
 Description=Albatross log daemon (albatross_log)
 After=syslog.target albatross_console.service
-Requires=albatross_console.service
+Requires=albatross_console.socket albatross_log.socket
```

```suggestion
Requires=albatross_log.socket
```

the albatross_log service does not depend on the albatross_console service